backend/src/test/java/com/ody/meeting/service/MeetingServiceTest.java

```diff
@@ -73,6 +73,34 @@ void findAllByMember() {
         );
     }
 
+    @DisplayName("내 약속 목록 조회 시 약속 시간이 현재 시간으로부터 24시간 이내인 약속부터 미래의 약속까지만 조회된다.")
```

Comment:
[제안]

포함 명시해주면 더 좋을 것 같아요!
```suggestion
    @DisplayName("내 약속 목록 조회 시 약속 시간이 현재 시간으로부터 24시간 포함 이내인 약속부터 미래의 약속까지만 조회된다.")
```

Reply:
[반영] [9b19489](https://github.com/woowacourse-teams/2024-ody/pull/302/commits/9b19489fc582d79171700366317bdb1f50c601fa)